inc/admin-pages/class-customer-edit-admin-page.php

```
@@ -96,6 +96,45 @@ public function hooks(): void {
+	/**
+	 * Handle secure deletion of customer meta fields.
+	 *
+	 * @since 2.0.11
+	 * @return void
+	 */
+	public function page_loaded() {
+
+		// Handle delete meta field action
+		if (isset($_GET['delete_meta_key']) && isset($_GET['_wpnonce'])) {
+
+			$meta_key = sanitize_key($_GET['delete_meta_key']);
```

PHPStan (2.1.17)

Function sanitize_key not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)